test/e2e/framework/test_context.go

```diff
@@ -399,7 +399,7 @@ func CreateGinkgoConfig() (types.SuiteConfig, types.ReporterConfig) {
 	// Randomize specs as well as suites
 	suiteConfig.RandomizeAllSpecs = true
 	// Disable skipped tests unless they are explicitly requested.
-	if len(suiteConfig.FocusStrings) == 0 && len(suiteConfig.SkipStrings) == 0 {
+	if len(suiteConfig.FocusStrings) == 0 && len(suiteConfig.SkipStrings) == 0 && suiteConfig.LabelFilter == "" {
 		suiteConfig.SkipStrings = []string{`\[Flaky\]|\[Feature:.+\]`}
```

Comment:
I didn't know about this behavior :/  , does not seem right to me but now explains why some jobs work wihtout setting skip or focus

cc: @BenTheElder 

/lgtm

Reply:
> I didn't know about this behavior :/

I knew about it, and hate it. But it has been here "forever" and has been carried forward because I never had the heart to remove it.

Reply:
Yeah, it's one of those things where we're going to break a lot of stuff that isn't setting it, it's also tricky to figure out because we often have N layers wrapping this binary.

I think it's a good intention to not run "bad" tests by default, but we've done a looot of work since then and it's ~arbitrary.

We should not carry this forward and this change is a good way to get out of that, thanks @pohly

Reply:
The default is still there. It just doesn't get applied anymore if one of --skip, --focus, or --label-filter is used explicitly.

Are you saying that it's now safe to remove this fallback completely?

Reply:
I have doubts removing this is safe, I vaguely remember there are a lot of jobs that does not set anything, and those will start picking all these flake and feature jobs

Reply:
Those jobs just need to set an equivilant default when being migrated?